HumanControls/MenuController.cpp

```diff
@@ -0,0 +1,83 @@
+#include "MenuController.h"
+
+MenuController::MenuController(unsigned int encoderPinClk, unsigned int encoderPinDt,
```

Comment:
Small formatting issue. Probably better to put each of these on their own line.

Reply:
Done